.github/workflows/ci.yaml

```diff
@@ -203,7 +203,11 @@ jobs:
           go-version: 1.20.5
 
       - name: Install prettier
-        run: npm install -g prettier
+        run: cd site && yarn --cache-folder=/tmp/yarn_pretty_cache install --prettier
+        with:
```

Comment:
this looks invalid,

Reply:
Yea, I am just going to stop trying to use the cache in this PR. Let's get things green on `main` then mess around with making it faster. I'm new to github actions

Reply:
It's blocking PRs.